src/main/java/net/jeremybrooks/knicker/Util.java

```diff
@@ -607,7 +607,12 @@ static String buildParamList(Map<String, String> params) throws KnickerException
 		for (String key : params.keySet()) {
 			try {
 				KnickerLogger.getLogger().log("Adding parameter key/value pair " + key + "=" + params.get(key));
-				sb.append(URLEncoder.encode(key, "UTF-8")).append('=').append(URLEncoder.encode(params.get(key), "UTF-8")).append('&');
+				String[] keyValueParts = params.get(key).split("\\,");
+				if( keyValueParts.length>1 ){
+					for(int i=0; i<keyValueParts.length; i++)
+						sb.append(URLEncoder.encode(key, "UTF-8")).append('=').append(URLEncoder.encode(keyValueParts[i], "UTF-8")).append('&');
+				}else
+					sb.append(URLEncoder.encode(key, "UTF-8")).append('=').append(URLEncoder.encode(params.get(key), "UTF-8")).append('&');
 			} catch (Exception e) {
```

Comment:
Using this in develop:
`				for (String part : params.get(key).split("\\,")) {
          sb.append(URLEncoder.encode(key, "UTF-8")).append('=').append(URLEncoder.encode(part, "UTF-8")).append('&');
        }`